src/components/ActionList/components/Item/Item.tsx

```diff
@@ -68,11 +69,15 @@ export function Item({
   );
 
   const badgeMarkup = badge && (
-    <span className={styles.BadgeWrapper}>
+    <span className={styles.Suffix}>
       <Badge status={badge.status}>{badge.content}</Badge>
     </span>
   );
 
+  const suffixMarkup = suffix && (
```

Comment:
What does this look like when both a badge and suffix element are passed in?

Reply:
Yeah I actually tested this scenario, but I should have provided screenshots to help reviewers.
![image](https://user-images.githubusercontent.com/1972567/92602420-1fb45580-f27c-11ea-90e6-8aa4a837ec7a.png)

In the end, my idea was to eventually deprecate/remove the badge, and the icon, and use more generic `prefix`/`suffix` allowing to add anything instead. But I didn't want to create any breaking change at the moment.

Reply:
Awesome, looks good and makes sense 👍